graylog2-server/src/main/java/org/graylog2/rest/resources/streams/StreamResource.java

```diff
@@ -397,7 +397,7 @@ private StreamResponse streamToResponse(Stream stream) {
             stream.getFields().get(StreamImpl.FIELD_CREATED_AT).toString(),
             stream.getDisabled(),
             stream.getStreamRules(),
-            stream.getAlertConditions(),
+            alertConditions,
```

Comment:
Not a big thing, but this may affect some code consuming the API: Alert conditions returned in here seem to have the type written in upper case letters, whereas before they used lower case. Alert conditions returned by `StreamAlertConditionResource` also use lower case for the type: https://github.com/Graylog2/graylog2-server/pull/2282/files#diff-51de2a31470f13c3ee6f29bc5129b989R157

I think the safest way to avoid any troubles is to simply convert the types in here to lower case, even if clients may already do so.